### PR TITLE
InstructorStudentListPageUiTest failing on live server #6620

### LINKED ISSUE #6620: InstructorStudentListPageUiTest failing on live server
v5.96

```
java.lang.AssertionError: expected:<CCSDetailsUiT.alice.tmms@gmail.tmt
benny.c.tmms@gmail.tmt
hugh.i.tmms@gmail.tmt
ivan.j.tmms@gmail.tmt
jack.k.tmms@gmail.tmt
CCSDetailsUiT.charlie.tmms@gmail.tmt
denny.c.tmms@gmail.tmt> but was:<CCSDetailsUiT.alice.tmms@gmail.tmt
benny.c.tmms@gmail.tmt
hugh.i.tmms@gmail.tmt
ivan.j.tmms@gmail.tmt
jack.k.tmms@gmail.tmt
CCSDetailsUiT.charlie.tmms@gmail.tmt
denny.c.tmms@gmail.tmt>
	at org.testng.AssertJUnit.assertEquals(AssertJUnit.java:101)
	at org.testng.AssertJUnit.assertEquals(AssertJUnit.java:108)
	at teammates.test.cases.BaseTestCase.assertEquals(BaseTestCase.java:133)
	at teammates.test.cases.browsertests.InstructorStudentListPageUiTest.testContent(InstructorStudentListPageUiTest.java:110)
	at teammates.test.cases.browsertests.InstructorStudentListPageUiTest.testAll(InstructorStudentListPageUiTest.java:52)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:659)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:845)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1153)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:771)
	at org.testng.TestRunner.run(TestRunner.java:621)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1199)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1124)
	at org.testng.TestNG.run(TestNG.java:1032)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)

```

### PR DESCRIPTION
**Pull request by xpdavid**
Fixes #6620. fixes the AppVeyor Failure in `master` branch.

**Outline of Solution**

> 1. Instead of using `verifyCopyEmailPopoverVisible`, we use `waitForCopyEmailPopoverVisible`.
> 1. Replace the line separator for value return by `exectueJsScript` (`return window.getSelection().toString();`) with `\n`, so that the line separator won't depends on the OS.

Build Pass on my AppVeyor Instance: https://ci.appveyor.com/project/xpdavid/teammates/build/1.0.6

### CODE REVIEW

**Comment by junhaoyap** [comment]
cc @wkurniawan07 for assigning someone(two maybe) who is for this area

**Comment by xpdavid** [comment]
@Neurrone Reminder, Pls review it. :P

File: src/test/java/teammates/test/pageobjects/InstructorStudentListPage.java [hunk 1]
```diff
@@ -51,13 +51,15 @@ public boolean isCopyEmailButtonVisible() {
         return copyEmailButton.isDisplayed();
     }
     
-    public boolean isCopyEmailPopoverVisible() {
+    public void waitForCopyEmailPopoverVisible() {
         String selector = "#copy-email-button + div.popover";
-        return isElementVisible(By.cssSelector(selector));
+        waitForElementPresence(By.cssSelector(selector));
```
Comment (JosephineKwa) [inline src/test/java/teammates/test/pageobjects/InstructorStudentListPage.java]: Let's use `waitForElementVisibility` instead as `waitForElementPresence` only checks if element is present is in DOM, regardless of whether it is visible or not.

**Comment by xpdavid** [comment]
@JosephineKwa Ready for review.

**Pull request merged**